common/djangoapps/student/urls.py

```diff
@@ -38,27 +47,18 @@
         url(r'^auto_auth$', 'auto_auth'),
     )
 
+# add all student.views url patterns
 urlpatterns = patterns(*urlpatterns)
 
-# password reset mixes student.views with django views
-urlpatterns += (
-    url(r'^password_reset/$', 'student.views.password_reset', name='password_reset'),
 
-    # Obsolete Django views for password resets
+# password reset django views (see above for password reset student.views)
+urlpatterns += patterns(
+    'django.contrib.auth.views',
 
     # TODO: Replace with Mako-ized views
```

Comment:
Reviewers: I presume this TODO is just about the single `password_reset_complete`?  Is there an advantage to this?  Is it just to keep the views consistent and together?  Other?  

I feel like I should remove the comment, or enhance to clarify why we want it.